mne/viz/_3d.py

```diff
@@ -637,6 +616,15 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
         If ``None``, creates a new 600x600 pixel figure with black background.
 
         .. versionadded:: 0.16
+    renderer : backends.Renderer | None
+        Rendering Scene in which to plot the alignment.
+        If ``None``, creates a new 600x600 pixel figure with black background.
+
+        .. versionadded:: 0.18
+    return_mayavi_figure : bool
+        If True (default), return mayavi figure.
+
+        .. versionadded:: 0.18
```

Comment:
I don't understand why there is this new kwarg. Why not always return it?

Reply:
(there are a couple of places this was added, so if you revert, please revert in all)

Reply:
I introduced it for deprecation reason at first but you're right, it's unnecessary, I'll remove it all.